torch/distributed/fsdp/wrap.py

```diff
@@ -296,26 +297,32 @@ class ParamExecOrderWrapPolicy:
     (also called non-recursive wrapping policy).
 
     The policy contains multiple wraps. Each wrap contains original parameters that will be executed together,
-    and the wrap transfers these parameters into one FlattenParameter. In both forward and the backward passes,
+    and the wrap transfers these parameters into one ``FlattenParameter``. In both forward and the backward passes,
     the sharded parameters in each wrap will be gathered just before these parameters are used in the passes.
     These parameters will then be reshaded once they have been used.
 
-    TODO (linjianma): For now, the parameters contained in each wrap of ParamExecOrderWrapPolicy
-    are the parameters in each wrap of the init_policy (a recursive wrapping policy).
+    TODO (linjianma): For now, the parameters contained in each wrap of ``ParamExecOrderWrapPolicy``
+    are the parameters in each wrap of the ``init_policy`` (a recursive wrapping policy).
     Later we will wrap parameters based on bucket size.
 
     Args:
-        init_policy (nn.Module):
+        init_policy (Callable):
```

Comment:
nit: iiuc, combined with tracing_config, init_policy is only used in construction time for module level wrapping, the module level reordering will be done before first iteration, right? if so, would you please help updating the comments to make it clear?